cap/config.py

```diff
@@ -771,4 +771,5 @@ def get_cms_stats_questionnaire_contacts():
         return {}
 
 
+SEND_MAIL_TO_PDF_FORUM = True
```

Comment:
Should be email or better list of emails. If it doesnt exist it wont be sent. Should get email/list from env var